agimus_controller_ros/agimus_controller_ros/simple_trajectory_publisher.py

```diff
@@ -6,6 +6,7 @@
 import pinocchio as pin
 from rclpy.qos import QoSProfile, DurabilityPolicy, ReliabilityPolicy
 import numpy as np
+from sensor_msgs.msg import JointState
 
 
 class MpcInputDummyPublisher(Node):
```

Comment:
This class name must reflect the file name:
`SimpleTrajectoryPublisher`
```suggestion
class SimpleTrajectoryPublisher(Node):
```